### PR TITLE
fixed bugs for consent to application , documents , check answers of …

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
…c100 application

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/alldocuments/template.njk [hunk 1]
```diff
@@ -10,7 +10,7 @@
   </h1>
   <div class="govuk-form-group">
     <fieldset class="govuk-fieldset">
-      <p class="govuk-body">{{caseNumber}} #{{userCase.id}}</p>
+      <p class="govuk-body">{{caseNumber}}  &nbsp#{{userCase.id}}</p>
```
Comment (vivek-sekhar) [inline src/main/steps/common/alldocuments/template.njk:13]: Remove hash and have space between caseNumber text and case id throughout the application

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/alldocuments/template.njk (hunk 1 above)
Comment (SasankAchyuta9) [reply src/main/steps/common/alldocuments/template.njk]: Done this.

File: src/main/steps/common/alldocuments/template.njk [hunk 2]
```diff
@@ -10,7 +10,7 @@
   </h1>
   <div class="govuk-form-group">
     <fieldset class="govuk-fieldset">
-      <p class="govuk-body">{{caseNumber}}  &nbsp#{{userCase.id}}</p>
+      <p class="govuk-body">{{caseNumber}}  &nbsp{{userCase.id}}</p>
```
[the remark this answers is not in the document]
Comment (SasankAchyuta9) [reply src/main/steps/common/alldocuments/template.njk:13]: Yeah removed extra spaces and given only one.

File: src/main/steps/prl-cases/dashboard/tabView.ts [hunk 3]
```diff
@@ -34,6 +34,7 @@ const caseStatusTranslation = {
   [State.CASE_ISSUED_TO_LOCAL_COURT]: 'caseIssued',
   [State.CASE_GATE_KEEPING]: 'caseGatekeeping',
   [State.CASE_SERVED]: 'caseServed',
+  [State.JUDICIAL_REVIEW]: 'caseGatekeeping',
```
[the remark this answers is not in the document]
Comment (SasankAchyuta9) [reply src/main/steps/prl-cases/dashboard/tabView.ts:37]: This judicial review is coming from back end.

File: src/main/app/case/definition.ts [hunk 4]
```diff
@@ -2042,6 +2042,7 @@ export const enum State {
   CASE_SERVED = 'PREPARE_FOR_HEARING_CONDUCT_HEARING',
   CASE_WITHDRAWN = 'CASE_WITHDRAWN',
   CASE_DELETED = 'REQUESTED_FOR_DELETION',
+  JUDICIAL_REVIEW = 'JUDICIAL_REVIEW',
```
Comment (vivek-sekhar) [inline src/main/app/case/definition.ts:2045]: I told you already please consider gate keeping as JR

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/app/case/definition.ts (hunk 4 above)
Comment (SasankAchyuta9) [reply src/main/app/case/definition.ts]: Yeah done this changes

**Review by vivek-sekhar — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**